test/client.js

```diff
@@ -57,6 +57,22 @@ describe('ServiceClient', () => {
         });
     });
 
+    it('should not throw error for empty response body', () => {
+        const client = new ServiceClient(clientOptions);
+        const originalBody = null;
+        requestStub.returns({
+            headers: {
+                'accept': 'text/plain'
```

Comment:
Here you return an `accept` header from the server instead adding it to request.